x-pack/plugins/security_solution/public/resolver/view/clickthrough.test.tsx

```diff
@@ -152,5 +152,26 @@ describe('Resolver, when analyzing a tree that has two related events for the or
         relatedEventButtons: 1,
       });
     });
+    describe('when the related events button is clicked', () => {
+      beforeEach(async () => {
+        const button = await simulator.resolveWrapper(() =>
+          simulator.processNodeRelatedEventButton(entityIDs.origin)
+        );
+        if (button) {
+          button.simulate('click');
```

Comment:
:grey_question: Mentioned above, but I had this as a public method on the simulator, like `simulator.clickNodeRelatedEventsButton(id)`